modules/memfault-firmware-sdk/config/memfault_metrics_heartbeat_extra.def

```diff
@@ -13,3 +15,9 @@ MEMFAULT_METRICS_KEY_DEFINE(Ncs_LteEdrxIntervalMsec, kMemfaultMetricType_Unsigne
 MEMFAULT_METRICS_KEY_DEFINE(Ncs_LteEdrxPtwMsec, kMemfaultMetricType_Unsigned)
 MEMFAULT_METRICS_KEY_DEFINE(Ncs_LteMode, kMemfaultMetricType_Unsigned)
 #endif /* CONFIG_MEMFAULT_NCS_LTE_METRICS */
+
+#ifdef CONFIG_MEMFAULT_NCS_BT_METRICS
+MEMFAULT_METRICS_KEY_DEFINE(Ncs_BtConnectionTime, kMemfaultMetricType_Timer)
+MEMFAULT_METRICS_KEY_DEFINE(Ncs_BtConnectionCount, kMemfaultMetricType_Unsigned)
+MEMFAULT_METRICS_KEY_DEFINE(Ncs_BtBondCount, kMemfaultMetricType_Unsigned)
```

Comment:
Consider collecting these statistics for each Bluetooth identity idepedently

Reply:
The design of the Memfault SDK is not really suitable for this.